css-positioning/fixed.css

```diff
@@ -0,0 +1,12 @@
+html {
+  font-size: 22px;
+}
+
+.page-title {
+  font-size: 3rem;
+  text-align: center;
+
+  /* your code here */
+  position: fixed;
+  left: 250px;
```

Comment:
😱 😱 😱 😱 

Is this to center the text? You can just set the element to be the full width of its containing block, it already has `text-align: center` applied to it.